src/java.base/windows/classes/sun/nio/fs/WindowsChannelFactory.java

```diff
@@ -291,20 +291,6 @@ private static FileDescriptor open(String pathForWindows,
             dwFlagsAndAttributes |= FILE_FLAG_OPEN_REPARSE_POINT;
         }
 
-        // permission check
-        if (pathToCheck != null) {
-            @SuppressWarnings("removal")
-            SecurityManager sm = System.getSecurityManager();
-            if (sm != null) {
-                if (flags.read)
-                    sm.checkRead(pathToCheck);
-                if (flags.write)
-                    sm.checkWrite(pathToCheck);
-                if (flags.deleteOnClose)
-                    sm.checkDelete(pathToCheck);
-            }
-        }
-
```

Comment:
I assume the pathToCheck parameter should be removed from newFileChannel and open methods.